packages/angular_devkit/build_angular/src/extract-i18n/index.ts

```diff
@@ -73,8 +75,12 @@ async function getSerializer(format: Format, sourceLocale: string, basePath: str
       const { SimpleJsonTranslationSerializer } =
         await import('@angular/localize/src/tools/src/extract/translation_files/json_translation_serializer');
 
-      // tslint:disable-next-line: no-any
       return new SimpleJsonTranslationSerializer(sourceLocale);
+    case Format.LegacyMigrate:
```

Comment:
I'm not totally sure whether this value is correct. The `./schema` import didn't work for me, but I assume it's generated from the `schema.json`?

Reply:
should be able to run `yarn build` to refresh the generated `.d.ts` files for the schemas.